akka-actor/src/main/scala/akka/actor/dungeon/TimerSchedulerImpl.scala

```diff
@@ -17,6 +17,8 @@ import akka.util.OptionVal
   final case class Timer(key: Any, msg: Any, repeat: Boolean, generation: Int, task: Cancellable)
   final case class TimerMsg(key: Any, generation: Int, owner: TimerSchedulerImpl)
     extends NoSerializationVerificationNeeded
+  final case class NotInfluenceReceiveTimeoutTimerMsg(key: Any, generation: Int, owner: TimerSchedulerImpl)
+    extends NoSerializationVerificationNeeded with NotInfluenceReceiveTimeout
```

Comment:
We look for `TimerMsg` in `aroundReceive` in `akka.actor.Timers` so it doesn't currently know about `NotInfluenceReceiveTimeoutTimerMsg`. Probably best to introduce a common `trait TimerMsg` and two implementations of it, or `NotInfluenceReceiveTimeoutTimerMsg extends TimerMsg` but then it shouldn't be a `case class` (probably fine with ordinary class anyway)

Reply:
Added `trait TimerMsg` and left the two implementations as `final case class` instead of just a plain class like `class NotInfluenceReceiveTimeoutTimerMsg(val key: Any, val generation: Int, val owner: TimerSchedulerImpl)` because I think the case class is a little more readable without the `val`s for each parameter.